lib/commands/debug/nodejs-debug.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const AbstractDebugFlow = require('./abstract-debug-flow');
+const helper = require('./helper');
+
+class NodejsDebugFlow extends AbstractDebugFlow {
+    static canHandle(runtime) {
+        return runtime.includes('node');
+    }
+
+    constructor({ skillInvocationInfo, waitForAttach, debugPort, token, skillId, debugRegion }) {
+        const skillFolderPath = path.join(process.cwd(), skillInvocationInfo.skillCodeFolderName);
+        const script = helper.getNodeDebugAdapterPath(skillFolderPath);
```

Comment:
should getNodeDebugAdapterPath be part of this class?

Reply:
moved to NodejsRunFlow class